xml/System/IntPtr.xml

```diff
@@ -951,7 +953,9 @@ For <xref:System.Numerics.IFloatingPoint%601>, this method matches the IEEE 754:
 
  Languages that do not support custom operators can call the <xref:System.IntPtr.Add%2A> method instead.
 
- The addition operation does not throw an exception if the result is too large to represent as a pointer on the specified platform. Instead, it is performed in an unchecked context.
+ The addition operation does not throw an exception if the result is too large to represent as a signed integer in the executing process. Instead, it is performed in an unchecked context.
+
+ In C# starting from version 11.0 and when targeting the .NET 7 or later runtime, this API is only accessible via reflection. The addition operator is directly recognized by the language and will follow the normal language behavior for addition operations, including overflowing in a `checked` context if the result is to large to represent.
```

Comment:
Not sure if its worth raising visibility on this distinction. For `operator +` this is a breaking change and contradictory to the immediate preceding statement. It will impact users who previously have code such as `IntPtr x = checked(value + int.MaxValue)` when they recompile.